### PR TITLE
Fixing Accessibility Isuues

### PR DESCRIPTION
**Pull request by helina42**
- Page titles :No issues found
- Image text alternatives :No issues found
- Text headings : Found Issues
- Color contrast :Found issues
- Resize : No issues found
- Interaction :Found issues
- Moving content : No issues found
- Multimedia : No issues found
- The basic structure of the page : No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <section class="container">
   <header class="toolbar">
     <a href="#">
-      <p>My logo</p>
+      <h2>My logo</h2>
```
Comment (helina42) [inline index.html:16]: Fix Text headings

File: index.html [hunk 2]
```diff
@@ -36,16 +36,16 @@ <h1 class="header">I’m Helina<br> Glad to see you!</h1>
           <ul class="icons">
             <li class="twitter"><a href="#"><img alt="Twitter Icon" src="./Icons/Twitter.png"></a></li>
             <li class="linkdin"><a href="#"><img alt="Linkedin Icon" src="./Icons/Linkedin icon.png"></a></li>
-            <li class="m"><a href="Media Icon"><img alt="medium" src="./Icons/M.png"></a></li>
-            <li class="cat"><a href="Cat Icon"><img alt="github" src="./Icons/cat.png"></a></li>
-            <li class="p"><a href="Peace Icon"><img alt="#" src="./Icons/Peace.png"></a></li>
+            <li class="m"><a href="#"><img alt="Medium Icon" src="./Icons/M.png"></a></li>
+            <li class="cat"><a href="#"><img alt="Github Icon" src="./Icons/cat.png"></a></li>
+            <li class="p"><a href="#"><img alt="Peace Icon" src="./Icons/Peace.png"></a></li>
```
Comment (helina42) [inline index.html]: Linked image with alternative text

File: index.html [hunk 3]
```diff
@@ -300,10 +300,12 @@ <h3 class="contact-head">Contact me</h3>
...
-        <input maxlength="30" type="text" name="name" placeholder="name" value="Yeremias" required>
-        <input type="email" name="email" placeholder="email" value="natajayanj14@gmail.com" required>
-        <textarea maxlength="500" type="text" name="message" placeholder="Write your message here" required
-          rows="8"></textarea>
+     
+        <input id ="name" maxlength="30" type="text" name="name" placeholder="name" value="Yeremias" required aria-label="name">
+
+        <input id ="email" type="email" name="email" placeholder="email" value="natajayanj14@gmail.com" required aria-label="email">
+
+        <textarea id ="message" maxlength="500" type="text" name="message" placeholder="Write your message here" required rows="8" aria-label="Write your message here"></textarea>
```
Comment (helina42) [inline index.html:308]: Interaction Label issues fixed

File: style.css [hunk 4]
```diff
@@ -9,7 +9,7 @@ body {
   box-sizing: border-box;
 }
 
-.toolbar p {
+.toolbar h2 {
```
Comment (helina42) [inline style.css:12]: Fix Text headings

File: style.css [hunk 5]
```diff
@@ -227,7 +227,7 @@ a {
   font-weight: 600;
   font-size: 13px;
   line-height: 16px;
-  color: #7a869a;
+  color: #4d5666;
```
Comment (helina42) [inline style.css:230]: Color contrast fix

**Review by helina42 — COMMENTED** [review]
The Accessibility Issues that are fixed are found on the codes

**Review by BregornOriginal — APPROVED** [review]
## Hi @helina42 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

giphy

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good Pull request description
✅ Your application looks great!

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**